packages/core/handlers/backend-utils.js

```diff
@@ -37,7 +37,42 @@ const createQueueWorker = (integrationClass) => {
     class QueueWorker extends Worker {
         async _run(params, context) {
             try {
-                const integrationInstance = new integrationClass();
+                let integrationInstance;
+
+                // Check if this is a webhook with integrationId
+                if (params.event === 'ON_WEBHOOK' && params.data?.integrationId) {
+                    // Load hydrated integration instance
+                    const { GetIntegrationInstance } = require('../integrations/use-cases/get-integration-instance');
```

Comment:
All this code will go away in the future when we introduce a decent IoC container.